<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="mainGridLayout">
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="textEditLayout">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Wy≈õlij</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Odbierz</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QGridLayout" name="settingsLayout">
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Flow control:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Stop bits:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="MappedComboBox&lt;QSerialPort::BaudRate&gt;" name="baudRateCB"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Data bits:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="MappedComboBox&lt;QSerialPort::Parity&gt;" name="parityCB"/>
        </item>
        <item row="4" column="1">
         <widget class="MappedComboBox&lt;SerialPort::FlowControl&gt;" name="flowControlCB"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Parity:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Baud rate:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="MappedComboBox&lt;QSerialPort::StopBits&gt;" name="stopBitsCB"/>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="terminatorCB"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Terminator:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="MappedComboBox&lt;QSerialPort::DataBits&gt;" name="dataBitsCB"/>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit">
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maxLength">
           <number>4</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="portSelectionLayout">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Select port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MappedComboBox&lt;QSerialPortInfo&gt;" name="serialPortInfoCB"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionRefresh_ports"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRefresh_ports">
   <property name="text">
    <string>Refresh ports</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MappedComboBox&lt;QSerialPort::DataBits&gt;</class>
   <extends>QComboBox</extends>
   <header>mappedcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>MappedComboBox&lt;QSerialPort::BaudRate&gt;</class>
   <extends>QComboBox</extends>
   <header>mappedcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>MappedComboBox&lt;QSerialPort::StopBits&gt;</class>
   <extends>QComboBox</extends>
   <header>mappedcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>MappedComboBox&lt;QSerialPort::Parity&gt;</class>
   <extends>QComboBox</extends>
   <header>mappedcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>MappedComboBox&lt;SerialPort::FlowControl&gt;</class>
   <extends>QComboBox</extends>
   <header>mappedcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>MappedComboBox&lt;QSerialPortInfo&gt;</class>
   <extends>QComboBox</extends>
   <header>mappedcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
